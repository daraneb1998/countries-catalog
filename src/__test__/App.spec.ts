import { describe, it, expect, beforeEach } from "vitest";

import { mount } from "@vue/test-utils";
import App from "@/App.vue";
import { createPinia, setActivePinia } from "pinia";

describe("App", () => {
  beforeEach(() => {
    setActivePinia(createPinia());
  });
  it("renders properly", () => {
    const wrapper = mount(App);
    expect(wrapper.text()).toMatchSnapshot();
  });
});
