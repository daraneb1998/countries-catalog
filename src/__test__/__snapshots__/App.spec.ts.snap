// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > renders properly 1`] = `"COUNTRIES CATALOG SORT NAMEASCENDINGDESCENDING Page 1 (25) of 0 PREVIOUS  NEXT"`;
